docs/workgraphs.md

```diff
@@ -1269,3 +1269,75 @@ NVIDIA performance is also quite underwhelming here.
 
 It's clear that native D3D12 drivers don't even try to optimize ExecuteIndirect at all :)
 Still, the ExecuteIndirect pattern of this demo makes EI look as bad as possible.
+
+### NVIDIA work graph example
+
+[This demo](https://github.com/NVIDIAGameWorks/donut_examples/blob/main/examples/work_graphs/)
+is another spin on the classic tile-deferred renderer.
+It's a little more realistic since the binner and classifier is emitting to Wave32 broadcast nodes,
+where each node is specialized in working on a specific material type. It's not using MaxDispatchGrid either,
+which should make scheduling very simple to deal with.
+
+#### RTX 4070 (native)
+
+GPU performance is extremely similar between workgraphs and plain compute. However, this is slightly concerning,
+since the compute shading path is [quite inefficient](https://github.com/NVIDIAGameWorks/donut_examples/blob/main/examples/work_graphs/deferred_shading.hlsl).
+It evaluates the materials in a big switch block, where the workgraph path can execute straight forward code.
```

Comment:
```suggestion
It evaluates the materials in a big switch block, where the workgraph path can execute straightforward code.
```

Reply:
I'm not particularly interested in drive by grammar review here.